Documentation/CRDs/Cluster/external-cluster/topology-for-external-mode.md

```diff
@@ -113,6 +113,13 @@ reclaimPolicy: Delete
 volumeBindingMode: WaitForFirstConsumer
 ```
 
+Update the rook-ceph-operator-config configmap data with the key value,
+
+[CSI_ENABLE_TOPOLOGY: "true"](https://github.com/rook/rook/blob/master/deploy/examples/operator.yaml#L509)
+
+[CSI_TOPOLOGY_DOMAIN_LABELS: "kubernetes.io/hostname,topology.kubernetes.io/zone,topology.rook.io/rack"](https://github.com/rook/rook/blob/master/deploy/examples/operator.yaml#L514)
```

Comment:
```suggestion
- `CSI_TOPOLOGY_DOMAIN_LABELS: "kubernetes.io/hostname,topology.kubernetes.io/zone,topology.rook.io/rack"`: Set the topology domain labels that the CSI driver will analyze on the nodes during scheduling.
```

Reply:
Why would you specify host, zone, and rack topology in the same value? Shouldn't you just set one topology type?

Reply:
Yes for the current example set to zone